SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java

```diff
@@ -763,6 +763,8 @@ protected void loadCampaignsFromJSON(JSONObject json, JSONObject settings) {
 
             Map<Integer, String> campaignsDownloaded = null;
 
+            boolean wasQAUser = (qaUser != null);
```

Comment:
I can change it but wanted to reflect that we are checking it was a QA user already before we might make it one after this refresh (on that same method)